.circleci/config.yml

```diff
@@ -623,12 +623,12 @@ jobs:
   test-wasm2js1:
     environment:
       EMTEST_SKIP_NODE_CANARY: "1"
-    executor: bionic
+    executor: focal
     steps:
       - run-tests-linux:
           test_targets: "wasm2js1"
   test-wasm64:
-    # We don't use `bionic` here since its tool old to run recent node versions:
+    # We don't use `focal` here since its tool old to run recent node versions:
```

Comment:
Should we change this comment and also line 665?

Reply:
Hmm.. I wonder if this is still true?   Maybe we can followup be removing this.

Reply:
Maybe revert these comment updates and we can look into removing these commends completely.   I wouldn't want to update the comments to something that is not true.